pdfrw/objects/pdfdict.py

```diff
@@ -101,8 +102,13 @@ class PdfDict(dict):
                     _stream=('stream', False),
                     )
 
+    whitespace = '\x00 \t\f'
+    delimiters = r'()<>{}[\]/%'
+    forbidden = whitespace + delimiters
+
     def __setitem__(self, name, value, setter=dict.__setitem__):
         assert name.startswith('/'), name
+        assert not any((c in self.forbidden) for c in name[1:]), name
```

Comment:
Instead of my original assertion and the new one:
  a)  Should we just assert it is a PdfName and be done with it?  (That way, it won't slow down most code as much, and people with bad code will learn to wrap their stuff in PdfNames.)
  b) We should probably use an exception rather than an assert (and that goes for a lot of my original asserts).